deploy.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
```

Comment:
Not a big deal but if we're adding deploy script to `news-search-api` i.e. completely separating it from `story-indexer`, does it make sense for `news-search-api` to continue referring to/have access to `story-indexer` config repo? Shouldn't that be completely separated as well?

Reply:
I agree that there should be a separate new-search-api config repo

Reply:
Cool, created `news-search-api-config` for this.